cob_hardware_config/robots/cob4-8/urdf/cob4-8.urdf.xacro

```diff
@@ -71,7 +71,7 @@
     <origin xyz="${head_cam_x} ${head_cam_y} ${head_cam_z}" rpy="${head_cam_roll} ${head_cam_pitch} ${head_cam_yaw}" />
   </xacro:usb_cam>
 
-  <xacro:sensorring name="sensorring" parent="head_3_link" active="true">
+  <xacro:sensorring name="sensorring" parent="head_2_link" active="true">
```

Comment:
if this is fixing the kinematic description of the head- sensorring-combo, we might need to apply this to all robots with such combo

Reply:
Hi @fmessmer,

we checked that the motions are the same in HW and in Rviz and they overlap (no gazebo though). I unfortunately can't verify that now as I am in the US for the next 4 months (I fixed this "head issue" before leaving). I will talk to my colleagues in Helsinki and see if somebody can do this check for you.

Reply:
Hi @fmessmer,
My colleague Jens Lundell promised to take over from here. I'll write you an email to put you two in contact.